src/features/collectables/NftDetailsScreen.tsx

```diff
@@ -22,6 +22,7 @@ import Trash from '../../assets/images/trash.svg'
 import TouchableOpacityBox from '../../components/TouchableOpacityBox'
 import InfoIcon from '../../assets/images/info.svg'
 import { ReAnimatedBox } from '../../components/AnimatedBox'
+import ArrowRight from '../../assets/images/arrowRight.svg'
 
 LogBox.ignoreLogs([
   'Non-serializable values were found in the navigation state',
```

Comment:
Seeing this one again, if we are going to ignore it maybe we just add it to the main ignoreLogs?

Reply:
Let me remove